sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/Aggregation.java

```diff
@@ -99,8 +136,8 @@ public static Aggregation explictBucketHistogram() {
    *
    * @param temporality Whether to report DELTA or CUMULATIVE metrics.
    */
-  public static Aggregation explictBucketHistogram(AggregationTemporality temporality) {
-    return explictBucketHistogram(
+  public static Aggregation explicitBucketHistogram(AggregationTemporality temporality) {
```

Comment:
do we need all of these options out of the box for v1? Could we limit the number of options we expose a bit more to start with? I see 4 methods for creating explicit bucket histograms which seems a tad excessive to start with. 

eg. `histogram()` and `explicitBucketHistogram()` are the same thing. Seems like we don't necessarily need both of them.

Reply:
`histogram` is specified as "the best possible histogram" and is allowed to change for the exponential in the future.

Regarding explicit bucket, I was matching the optional parameters of the spec.